### PR TITLE
Scott hashing

### PR DESCRIPTION
**Pull request by stdavis**
All tests are passing now.

### CODE REVIEW

File: src/forklift/core.py [hunk 1]
```diff
@@ -188,12 +188,12 @@ def _create_destination_data(crate):
...
 
-    shape_type = arcpy.Describe(crate.source).shapeType.upper()
+    source_describe = arcpy.Describe(crate.source)
     arcpy.CreateFeatureclass_management(crate.destination_workspace,
                                         crate.destination_name,
-                                        shape_type,
+                                        source_describe.shapeType.upper(),
                                         crate.source,
-                                        spatial_reference=crate.destination_coordinate_system)
+                                        spatial_reference=crate.destination_coordinate_system or source_describe.spatialReference)
```
Comment (steveoh) [inline src/forklift/core.py]: should this happen automatically on the crate?

File: src/forklift/core.py [hunk 2]
```diff
@@ -136,7 +136,7 @@ def remove_temp_table(table):
                                                                  crate.name,
                                                                  source_describe.shapeType.upper(),
                                                                  crate.source,
-                                                                 spatial_reference=source_describe.spatialReference)
+                                                                 spatial_reference=source_describe.spatialReference).getOutput(0)
```
Comment (steveoh) [inline src/forklift/core.py]: can we destructure this? 

[Result outputs can also be accessed by index, so result.getOutput(0) and result[0] are equivalent.](http://resources.arcgis.com/en/help/main/10.1/index.html#//018z00000046000000)

**Comment by steveoh** [comment]
✨ ✨ ✨ ✨ ✨ ✨ ✨ ✨ ✨ ✨ ✨ ✨ ✨ ✨ ✨ ✨ ✨ ✨ ✨ ✨ ✨ ✨ ✨ ✨ 
```
Name                     Stmts   Miss     Cover   Missing
---------------------------------------------------------
forklift\arcgis.py          65      3    95.38%   48, 96-98
forklift\cli.py            159      8    94.97%   102, 161-166, 173, 220-223
forklift\core.py           220     25    88.64%   45, 66-68, 115, 137-157, 266-268, 305-308, 358, 440-441
forklift\lift.py           127     26    79.53%   44-45, 131-152, 158, 176-177, 213-215, 228-229
forklift\models.py         115      1    99.13%   79
---------------------------------------------------------
TOTAL                      775     63    91.87%

5 files skipped due to complete coverage.
```

**Comment by steveoh** [comment]
refs #81

File: src/forklift/core.py (hunk 1 above)
Comment (steveoh) [inline src/forklift/core.py]: i didn't think about the call to describe, let's leave it for not until we work on all the describing.

**Comment by stdavis** [comment]
@steveoh: What's the next steps on this? Is someone working on fleshing out tests for the new code? I'm happy to help, just looking for direction.

**Comment by steveoh** [comment]
@keatonwalker isn't in today so I'm not sure where he's at with new tests. Let me stop messing around with atom and fool around with this pr a bit more before I merging to master. Go ahead and start with new tests if you are up for it.

**Pull request merged**